src/lib/protocols/present-proof/__tests__/ProofState.test.ts

```diff
@@ -0,0 +1,13 @@
+import { ProofState } from '../ProofState';
+
+describe('ProofState', () => {
+  test('state matches Present Proof 1.0 (RFC 0037) state value', () => {
+    expect(ProofState.ProposalSent).toBe('proposal-sent');
+    expect(ProofState.ProposalReceived).toBe('proposal-received');
+    expect(ProofState.RequestSent).toBe('request-sent');
+    expect(ProofState.RequestReceived).toBe('request-received');
+    expect(ProofState.PresentationSent).toBe('presentation-sent');
+    expect(ProofState.PresentationReceived).toBe('presentation-received');
+    expect(ProofState.Done).toBe('done');
+  });
+});
```

Comment:
@jakubkoci I added tests for each of the state object to test for the string value. The other tests now use the enum value. This is to keep the tests separated

Reply:
Good 👍